.aws/production/taskDefinition.json

```diff
@@ -0,0 +1,128 @@
+{
```

Comment:
@lreesby adding you to take a look at the actions config files since you set that up for the webapp and understand it better than I do.